app/pods/publications/publication/publication-activities/index/template.hbs

```diff
@@ -0,0 +1,79 @@
+<Auk::Panel class="auk-u-mt-4">
+  <Auk::Panel::Header>
+    <Auk::Toolbar as |Toolbar|>
+      <Toolbar.Group @position="left" as |Group|>
+        <Group.Item>
+          <h4 class="auk-panel__title">
+            {{t "documents"}}
+          </h4>
+        </Group.Item>
+      </Toolbar.Group>
+      <Toolbar.Group @position="right" as |Group|>
+        {{#unless this.isViaCouncilOfMinisters}}
+          <Group.Item>
+            <Auk::Button
+              @skin="primary"
+              @icon="plus"
+              {{on "click" this.openReferenceDocumentUploadModal}}
+            >
+              {{t "document-add"}}
+            </Auk::Button>
+          </Group.Item>
+        {{/unless}}
+      </Toolbar.Group>
+    </Auk::Toolbar>
+  </Auk::Panel::Header>
+  <Auk::Panel::Body class="auk-u-bg-alt">
+    <Utils::Generic::Timeline::Timeline @model={{@model}} as |Timeline|>
+      <Timeline.Event @badgeIcon="check" @badgeSkin="success" as |event|>
+        {{#if event.isRequest}}
+          {{!Aanvraag}}
+          <Auk::Panel>
+            <Auk::Panel::Header>
```

Comment:
Hier zou een `Auk::AccordionPanel` moeten komen. De huidige versie daarvan laat niet toe dat een titel meer is dan een string. Is het OK dat ik hier de Auk-component aanpas om naast een @title-attribute ook een :title-block te aanvaarden?

Reply:
@mdv-professional Ik heb van @TaneeG vorige week quasi dezelfde vraag gekregen. Binnen `feature/KAS-3106_Vertalingen_herwerken` (#1193) heb ik daarvoor `accordion-panel` en `panel` uitgebreid met een `@subtitle` argument.

![CleanShot 2022-02-14 at 11 11 15@2x](https://user-images.githubusercontent.com/18174827/153844745-1b24ecda-b08c-4d36-bfdb-c4f5ab3910c8.png)

Wat lijkt je het meest makkelijk om die changes bij te krijgen? Wacht je tot de branch gemerged is naar `development` of geef ik de nodige commits door zodat je ze desnoods kan cherry-picken?

Reply:
OK. Ik heb ze gekopieerd.